index.js

```diff
@@ -1,3 +1,5 @@
-require('dotenv').config()
+import { config } from 'dotenv'
 
-require('./lib/server')
+import './lib/server.js'
+
+config()
```

Comment:
doesnt seem right? 

I think first config then import the server makes more sense